Lib/asyncio/graph.py

```diff
@@ -17,7 +17,7 @@
 )
 
 if False:  # for type checkers
```

Comment:
We can unlazify this import now since `io` gets imported anyway if you import `asyncio.graph` (the path is `concurrent.futures -> logging -> io`).